WORKSPACE

```diff
@@ -0,0 +1,14 @@
+workspace(name = "com_google_api_gax_java")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("//:repository_rules.bzl", "com_google_api_gax_java_properties")
+
+com_google_api_gax_java_properties(
+    name = "com_google_api_gax_java_properties",
+    file = "//:dependencies.properties",
+)
+
+load("//:repositories.bzl", "com_google_api_gax_java_repositories")
```

Comment:
Is it normal to put the load statement here instead of all at the top?

Reply:
The best practices say that they should be put at the top, but it is not always possible (and sometimes it is better to put load statements lower for better readability, to not mix different languages builds togethere). Here it would not work, because `:repositories.bzl` imports stuff generated by the previous repository rule `com_google_api_gax_java_properties`. Note `com_google_api_gax_java_properties` is a repository rule (not same thing as a regular rule).